src/Resources/public/core.js

```diff
@@ -941,20 +941,24 @@ var Backend =
 				alert('Could not find the SimpleModal frame');
 				return;
 			}
-			ul = frm.document.getElementById(opt.id);
+			ul = frm.document.getElementById('tl_select');
```

Comment:
That's possible of course, but it does not really help. Do we want to support `opt.id` in Contao 5 as well? Then we should encourage people to provide it. Or do we want to use `tl_select` in Contao 5? Then we should add a deprecation notice if someone uses `opt.id`.